packages/atomic/src/pages/examples/csp.html

```diff
@@ -0,0 +1,388 @@
+<!doctype html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Coveo Atomic</title>
+    <link rel="icon" href="data:," />
+    <link rel="stylesheet" href="/themes/coveo.css" />
+    <script type="module" nonce="1234567890">
+      import {setNonce} from '/build/atomic.esm.js';
+
+      setNonce('1234567890');
```

Comment:
We change how we import Atomic. If we call setOnce after we imported atomic in a different script, we'll still have some CSP error.
From what I experienced, if we do not call `setOnce` in the same microtask as the import of atomic, some style tag will slip through and end up without nonce.